service/http.proxy/src/test/java/org/kaazing/gateway/service/http/proxy/HttpProxyStreamingIT.java

```diff
@@ -52,8 +57,13 @@
         }
     };
 
+    TestRule trace = new MethodExecutionTrace();
+    TestRule timeoutRule = new DisableOnDebug(Timeout.builder().withTimeout(10, SECONDS)
+            .withLookingForStuckThread(true).build());
```

Comment:
The javadoc for withLookingForStuckThread claims it prints the stack trace (assuming that's the same thing as a backtrace): "Specifies whether to look for a stuck thread. If a timeout occurs and this feature is enabled, the rule will look for a thread that appears to be stuck and dump its backtrace."